tests/integrations/test_pipeline_registration.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from garden_ai.app.pipeline import register
```

Comment:
I think (but don't totally remember why) it's better to test the cli commands with the `typer.testing.CliRunner` class like in the `tests/test_cli.py` than to exercise the underlying functions directly